Ambit/Source/Ambit/Actors/Spawners/SpawnerBase.cpp

```diff
@@ -280,48 +277,48 @@ void ASpawnerBase::SpawnActorsAtTransforms(
         TArray<bool> OriginalGenerateOverlapEvents;
         // Set GenerateOverlapEvents to true while Play mode is active
         AmbitSpawnerCollisionHelpers::
```

Comment:
This is unusual wrapping, even if we were limiting to 80 chars. If we're limiting to 120 chars, lines 279, 280, and 291 should all fit on a single line. But even if were were limiting to 80 chars I would expect lines 279 and 280 to be on a single line. Is there some sort of formatting rule (besides line length) at play here?